ir/src/main/scala/org/scalajs/core/ir/Utils.scala

```diff
@@ -66,33 +67,39 @@ object Utils {
     // scalastyle:on return
   }
 
-  private def createEscapeJSString(str: String): String = {
+  def printEscapeJS(str: String, out: java.io.Writer): Unit = {
```

Comment:
With the refactoring prompted by the introduction of this method, I think the comment in `escapeJS` should be moved in `printEscapeJS` instead.